package.json

```
@@ -48,6 +48,7 @@
     "remark-import-partial": "^0.0.2",
     "remark-math": "^3.0.1",
     "remark-remove-comments": "^0.2.0",
+    "require-glob": "^4.1.0",
     "url-loader": "^4.1.1",
```

With `core pack enable` this shouldn't be needed.
If we commit this, we also need to update the yarn.lock file

```suggestion
    "remark-remove-comments": "^0.2.0",
    "url-loader": "^4.1.1",
```